src/renderer/constants.ts

```diff
@@ -1,6 +1,16 @@
 import * as path from 'path';
 
 export const USER_DATA_PATH = window.ElectronFiddle.appPaths.userData;
+export const ELECTRON_DOWNLOAD_PATH = path.join(
+  USER_DATA_PATH,
+  '.electron-bin',
+  'zips',
+);
+export const ELECTRON_INSTALL_PATH = path.join(
+  USER_DATA_PATH,
+  '.electron-bin',
```

Comment:
Oh yeah! However, we won't be still able to use the existing downloaded versions as `fiddle-core` keeps them in a zip file and also uses a different naming convention of files.

Reply:
That's a big breaking change for users (especially if they have 10+ Electron versions downloaded), and I don't think I'm super comfortable releasing this as-is.

Is there a way we can either:
1. Run a startup script that moves Fiddle scripts to a format that fiddle-core understands?
2. Make changes to Fiddle Core so it understands folders rather than zips?

Reply:
I agree with Erick, this is a pretty major change for existing Fiddle users. Of the two options, could we modify fiddle-core to understand both zip and folder structures? I feel like that would be the more future-proof option

Reply:
I also agree with the second.
The first one will be more costly, if the user downloads very many versions of the electron, the migration script will be very slow to execute and will predictably have a lot of error handling logic. And with a lot of unit tests to go along with it

Reply:
One more thing to consider: when using fiddle to load a locally built electron, the electron is a folder, not a zip file.

<img width="923" alt="image" src="https://user-images.githubusercontent.com/8198408/186296298-d19e9a0b-e98a-4c2a-8fbe-d552df013b42.png">